xdg-desktop-portal-wlr.5.scd

```diff
@@ -71,6 +71,12 @@ These options need to be placed under the **[screencast]** section.
 	- simple, dmenu: xdpw will launch the chooser given by **chooser_cmd**. For more details
 	  see **OUTPUT CHOOSER**.
 
+**force_mod_linear** = _bool_
```

Comment:
I feel like this should be a debug option, and nothing more. Modifiers should work correctly out of the box on multi-GPU setups too.

Reply:
This option is meant of `implicit modifier` only! Should I clarify this in the text? The other option would either be using a env-variable, or checking if multiple GPU's are available and using linear for implicit internally?

Reply:
In case this should be demoted to a debug option, should this be done via build or env variable?

Reply:
Marked this option as experimental, so we can remove it from the config or replace it with an environment variable later.